app/tests/retina_api_tests/helpers.py

```diff
@@ -82,404 +65,3 @@ def create_datastructures_data(archive_pars=None):
         oct_obs_registration,
         oct_obs_registration_aus,
     )
-
-
-def batch_test_image_endpoint_redirects(test_class):
-    for image_type, reverse_name in (
-        ("thumb", "retina:image-thumbnail"),
-        ("original", "retina:image-numpy"),
-    ):
-        (
-            test_redirect_no_perm,
-            test_redirect,
-            test_redirect_australia,
-            test_redirect_kappa,
-            test_redirect_oct,
-        ) = create_image_test_method(image_type, reverse_name)
-        test_redirect_no_perm.__name__ = (
-            f"test_image_{image_type}_redirect_no_perm"
-        )
-        test_redirect.__name__ = f"test_image_{image_type}_redirect_rotterdam"
-        test_redirect_kappa.__name__ = (
-            f"test_image_{image_type}_redirect_kappa"
-        )
-        test_redirect_australia.__name__ = (
-            f"test_image_{image_type}_redirect_australia"
-        )
-        test_redirect_oct.__name__ = f"test_image_{image_type}_redirect_oct"
-        setattr(
-            test_class, test_redirect_no_perm.__name__, test_redirect_no_perm
-        )
-        setattr(test_class, test_redirect.__name__, test_redirect)
-        setattr(
-            test_class,
-            test_redirect_australia.__name__,
-            test_redirect_australia,
-        )
-        setattr(test_class, test_redirect_kappa.__name__, test_redirect_kappa)
-        setattr(test_class, test_redirect_oct.__name__, test_redirect_oct)
-
-
-def create_image_test_method(image_type, reverse_name):
-    def test_redirect_no_perm(self, client):
-        ds = create_some_datastructure_data()
-        url = reverse(
-            "retina:api:image-api-view",
-            args=[
-                image_type,
-                ds["patient"].name,
-                ds["study"].name,
-                ds["image_cf"].name,
-                "default",
-            ],
-        )
-
-        user, _ = get_retina_user_with_token()
-        client.force_login(user=user)
-        response = client.get(url, follow=True)
-        expected_redirect_url = django_reverse(
-            reverse_name, args=[ds["image_cf"].id]
-        )
-        assert status.HTTP_302_FOUND == response.redirect_chain[0][1]
-        assert expected_redirect_url == response.redirect_chain[0][0]
-        assert status.HTTP_403_FORBIDDEN == response.status_code
-
-    def test_redirect(self, client):
-        ds = create_some_datastructure_data()
-        url = reverse(
-            "retina:api:image-api-view",
-            args=[
-                image_type,
-                ds["patient"].name,
-                ds["study"].name,
-                ds["image_cf"].name,
-                "default",
-            ],
-        )
-
-        user, _ = get_retina_user_with_token()
-        client.force_login(user=user)
-        ds["image_cf"].permit_viewing_by_retina_users()
```

Comment:
Is the function `permit_viewing_by_retina_users()` required anymore https://github.com/comic/grand-challenge.org/blob/ba63026bc1024eac7233b6a677ba1da780387670/app/grandchallenge/cases/models.py#L447

Reply:
Actually still seems to be used here https://github.com/comic/grand-challenge.org/blob/8ef178ef1b0f135ef3eb85818c84cfc4e6538f53/app/grandchallenge/retina_importers/views.py#L162